LocalSettings.php

```diff
@@ -1198,6 +1198,10 @@
 	'wgEmbedVideoRequireConsent' => [
 		'default' => true,
 	],
+	'wgEmbedVideoFetchExternalThumbnails' => [
+		'default' => true,
+		'comprehensibleinputwiki' => false,
```

Comment:
Can you remove this line as https://github.com/miraheze/mw-config/pull/5091 will allow this to be changed in ManageWiki (but relies on this PR to set the default value)?

Reply:
Sure.